### ISSUE TITLE
Possible issue attribute value 0

### ISSUE THREAD

**Issue by RetrospectiveRotations** [josuebrunel/pysxm#20]
Hi

In line 89 of pysxm.py, you have

```python
                if not attr:
                    continue
```

Which causes attributes with value 0 in your sequence not being added to the output XML. E.g., suppose I want to print M from some class to XML,  then

```python
        self.M = 0 
```

will not be printed, while

```python
    self.M = 0.1
```

will appear in the XML.

I fixed this by adding & (attr != 0), however you might have a more elegant solution.

Kind regards,

Peter

**Comment by josuebrunel** [comment]
Hi @Muiroe ,
Thanks for the issue and good catch. Do you mind submitting a patch ( Pull Request ) ? I'll merge it as soon as I can

**Comment by RetrospectiveRotations** [comment]
You mind altering that single line of code yourself? Im not exactly pro with Git, yet.

**Comment by josuebrunel** [comment]
Ok no worries, will do it. 
Thanks again

**Comment by josuebrunel** [comment]
@Muiroe the issue is fixed and a new version (v1.4) is released. 
Have a good one
